modules/hcp-support-matrix.adoc

```diff
@@ -0,0 +1,144 @@
+
+.Hosted cluster versions that can be created by {mce-short} versions
+[cols="5",options="header"]
+|===
+|Hosted cluster version  |{mce-short} 2.4 |{mce-short} 2.5 |{mce-short} 2.6 |{mce-short} 2.7
+
+|4.14
+|Yes
+|Yes
+|Yes
+|Yes
+
+|4.15
+|No
+|Yes
+|Yes
+|Yes
+
+|4.16
+|No
+|No
+|Yes
+|Yes
+
+|4.17
+|No
+|No
+|No
+|Yes
+|===
+
+When you update to another version of {mce-short}, your hosted cluster can continue to run if the HyperShift Operator that is included in the version of {mce-short} supports the hosted cluster version. The following table shows the affects of {mce-short} updates on various versions of hosted clusters:
+
+.Affects of {mce-short} updates on hosted clusters
+[cols="4",options="header"]
+|===
+|Hosted cluster version| Update from {mce-short} 2.4 to 2.5 |Update from {mce-short} 2.5 to 2.6 |Update from {mce-short} 2.6 to 2.7
+
+|{product-title} 4.14
+|No affect, hosted cluster continues to run
+|No affect, hosted cluster continues to run
+|No affect, hosted cluster continues to run
+
+|{product-title} 4.15
+|
+|No affect, hosted cluster continues to run
+|No affect, hosted cluster continues to run
+
+|{product-title} 4.16
+|
+|
+|No affect, hosted cluster continues to run
+|===
+
+For example, {mce-short} works with both {product-title} 4.14 and 4.15 clusters. If you have a {product-title} 4.14 hosted cluster on the management cluster and you update from {mce-short} 2.4 to 2.5, the hosted cluster can continue to run.
+
+The following table indicates which {product-title} versions are supported for each platform of {hcp}. In the table, management cluster version refers to the {product-title} version where the {mce-short} is enabled:
+
+.Required {product-title} versions for platforms
+[cols="3",options="header"]
+|===
+|Platform |Management cluster version |Hosted cluster version
+
+|{aws-full}
+|4.12 - 4.17
+|4.14 - 4.17
+
+|{ibm-power-title}
+|4.15 - 4.17
+|4.15 - 4.17
+
+|{ibm-z-title}
+|4.15 - 4.17
+|4.15 - 4.17
+
+|{VirtProductName}
+|4.14 - 4.17
+|4.14 - 4.17
+
+|Bare metal
+|4.14 - 4.17
+|4.14 - 4.17
+
+|Non-bare-metal agent machines (Technology Preview)
+|4.16 - 4.17
+|4.16 - 4.17
+|===
+
+The following table indicates availability for {hcp} features as of version 4.17:
+
+[cols="2",options="header"]
+|===
+|Feature |Support level
+
+|{hcp-capital} on bare metal in a disconnected environment
+|Generally Available
+
+|{hcp-capital} on {VirtProductName} in a disconnected environment
+|Generally Available
+
+|{hcp-capital} on {ibm-z-title} in a disconnected environment
+|Technology Preview
+
+|Custom taints and tolerations for {hcp} on {VirtProductName}
+|Technology Preview
+
+|NVIDIA GPU devices on {hcp} for {VirtProductName}
+|Technology Preview
+
+|{hcp-capital} for ARM64 {product-title} clusters on {aws-short}
+|Generally Available
+|===
```

Comment:
Tiny nit, can we order by status (i.e., all the GAed features first then TP). An alternative would be to only highlight tech-preview features. For example, "the following features are still TP" if we want to avoid chasing existing GAed features.

Reply:
Makes sense. I like the idea of listing just the TP features.